src/gt4py/storage/storage.py

```diff
@@ -262,6 +269,34 @@ def host_to_device(self, force=False):
     def device_to_host(self, force=False):
         pass
 
+    if dace is not None:
+
+        def __descriptor__(self) -> "dace.data.Array":
```

Comment:
@gronerl I realized we can do this on methods elsewhere that are only relevant to dace.